### DISCUSSION TITLE
How can I update the price of an item ONLY for the current cart

### DISCUSSION THREAD

**Discussion by JoeMatkin** [swellstores/community#337] [Help] [answered]
if I try to update a cart item price using

```javascript
await swell.cart.updateItem('7d51p8ce72f5542e009fa4c8', {
  quantity: 2
})
``` 
I get the error: Error: You are not allowed to update `items.price`

I only want to update the cart item for the current cart/session.

For context, I pull in a price for a diamond from an api (based on filter selection) and instead of creating another product and adding that into cart I want to merge the price with the "ring mount" price.

Thanks

**Comment by sags95** [comment, marked as answer]
Hey @JoeMatkin, for updating the price of an item in the cart you would have to do this through the Backend API by updating the cart items: https://developers.swell.is/backend-api/carts/update-a-cart
